docs/monitoring/understanding-dashboards.md

```diff
@@ -0,0 +1,460 @@
+
+- **execution**: time taken to execute the transactions in the block
+- **validation**: time taken to compute a new state root and compare it to the one that arrived in the block
+- **commit**: time taken to write the new block to the chain data
+- **account read**: time taken to access account information from the state trie
+- **account update**: time taken to update a leaf in the state trie
+- **account hash**: time taken to generate a hash of an account's data
+- **account commit**: time taken to write new account data into the state trie
+- **storage read**: time taken to access smart contract storage data from the storage trie
+- **storage update**: time taken to change a piece of smart contract storage data in the storage trie
+- **storage hash**: time taken to generate a new hash for modified smart contract storage data
+- **storage commit**: time taken to write modified smart contract storage data to the storage trie. 
+- **snapshot account read**: time taken to read account data from a snapshot
+- **snapshot storage read**: time taken to read storage data from a  snapshot
+- **snapshot commit**: time taken to write data to a snapshot
+
+![The block processing panel](/images/docs/grafana/block-processing.png)
+
+#### Transaction processing
+
+The transaction processing panel tracks the time taken to complete the various tasks involved in processing each block, measured as a mean rate of events per second:
+
+- **known**: rate of new transactions arriving at the node that are ignored because the local node already knows about them.
+- **valid**: rate that node marks known transactions as valid
+- **invalid**: rate that node marks known transactions as invalid
+- **underpriced**: rate that node marks transactions paying insufficient gas as invalid
+- **executable discard**: rate that valid transactions are dropped from the transaction pool, e.g. because it is already known.
+- **executable replace**: rate that valid transactions are replaced with a new one from same sender with same nonce but higher gas
+- **executable ratelimit**: rate that valid transactions are dropped due to rate-limiting
+- **executable nofunds**: rate that valid transations are dropped due to running out of ETH to pay gas
+- **gapped discard**: rate that queued transactions are discarded from the transaction pool
+- **gapped replace**: rate that queued transactions are replaced with a new one from same sender with same nonce but higher gas
+- **gapped ratelimit**: rate that queued transactions are dropped due to rate limiting
+- **gapped nofunds**: rate that queued transactions are dropped due to running out of ETH to pay gas
+
+![The tx processing panel](/images/docs/grafana/tx-processing.png)
+
+#### Block propagation
+
+Block propagation metrics track the rate that the local node hears about, receives and broadcasts blocks. This includes:
```

Comment:
We need to add a note that block propagation is totally disabled after The Merge. So all numbers are expected to be zero.

Reply:
ah yes, of course - thanks.
```suggestion

<Note>
Block propagation was disabled in Geth at The Merge. Block propagation is now the responsibility of the consensus client.  Included here for archival interest.
</Note>
 
Block propagation metrics track the rate that the local node hears about, receives and broadcasts blocks. This includes:
```